pkg/redis/redis.go

```diff
@@ -0,0 +1,69 @@
+package redis
+
+import (
+	"context"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/mediocregopher/radix/v3"
+	"github.com/virtual-vgo/vvgo/pkg/log"
+	"github.com/virtual-vgo/vvgo/pkg/tracing"
+)
+
+type Client struct {
+	config Config
+	pool   *radix.Pool
+}
+
+type Config struct {
+	Network  string `default:"tcp"`
+	Address  string `default:"localhost:6379"`
+	PoolSize int    `split_words:"true" default:"10"`
+}
+
+var client *Client
```

Comment:
Should this be attached to the main API Server struct, instead of a package global?

This is probably fine, but I do want to make this point about our overall structure.

Reply:
I thought about including it in the API server struct, but then... would we ever need more than 1 redis client? I'm not sure...